src/main/java/core/basesyntax/files/CsvFileWriter.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class CsvFileWriter implements FileWriter {
+    @Override
+    public void writeToFile(String report, String toFileName) {
+        try {
+            Files.write(Path.of(toFileName), report.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file, ", e);
```

Comment:
same mistake second time( Check all similar places in project 
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/d8859e87-60ac-4ff2-bce1-3b16c84abe37)